modules/unsupported/geojson-core/src/test/java/org/geotools/data/geojson/GeoJSONReaderTest.java

```diff
@@ -177,7 +177,7 @@ public void testFeatureCollectionParser() throws Exception {
         f = DataUtilities.first(features);
 
         assertNotNull(f);
-        assertEquals("features.0", f.getID());
+        assertEquals("feature.0", f.getID());
```

Comment:
I don't like that these have changed, is there a reason for the loss of the `s`?

Reply:
Originally changed to match GeoJSON in the tests.  I have gone ahead and added the s to JSON and reverted the asserts.